utils/data_preprocessing.py

```diff
@@ -201,9 +204,15 @@ def load_data(dir: str, unwanted_keywords: Set[str]):
 
 def load_data_from_frontend(docs: dict):
     filenames = [doc["id"] for doc in docs]
-    overall = [doc["content"].split() for doc in docs]
+    texts = [doc["content"] for doc in docs]
+    langs = [doc["language"] for doc in docs]
+    lang = 0
+    for i in langs:
```

Comment:
I think you made a mistake here?

overall = [clean(text, lang) for text, lang in zip(texts, langs)]

but the content is already clean when it comes from the frontend so no need to clean it again.

Reply:
doc["content"] is only giving us the content of document from solr 
so basically
overall = [doc["content"].split() for doc in docs] 
is only returning us the splitted words from content and we need to clean that first.

Reply:
The content is Solr is clean. I perform a cleaning before including the content into Solr

Reply:
can you please try to print the content from Solr ? 
and compare with the cleaned ones

Reply:
It is the same, I am cleaning with your clean function.

https://github.com/AMOS-5/infinitag/blob/master/backend/solr/docstorage.py#L81